misc/diagram-util/src/main/java/org/ballerinalang/diagramutil/connector/models/connector/reftypes/RefType.java

```diff
@@ -61,4 +62,14 @@ public RefType clone() {
         }
     }
 
+    public String getHashCode() {
+        return hashCode;
+    }
+
+    public String getTypeName() {
+        return typeName;
+    }
```

Comment:
Either use public `typeName` without getter or have a getter with private variable

Reply:
Same applies to `hashCode` as well